File: babylonian_square_root/babylonian_square_root.cs
using System;

public class babylonian_square_root
{
	public static double squareRoot(double num)
	{
		x = num;
		y = 1;
		e = 0.00000001;

		while (x - y > e)
		{
			x = (x + y) / 2;
			y = num / x;
		}
		return x;
	}

	static void Main(string[] args)
	{
		Console.WriteLine(squareRoot(15));
	}
}

File: babylonian_square_root/babylonian_square_root.java
public class babylonian_square_root {

	public static double squareRoot(double num) {
		double x = num;
		double y = 1;
		double e = 0.00000001;

		while (x - y > e) {
			x = (x + y) / 2;
			y = num / x;
		}
		return x;
	}

	public static void main(String[] args) {
		System.out.println(squareRoot(15));
	}
}
